### ISSUE TITLE
Drop-path implementation

### ISSUE THREAD

**Issue by mirosval** [snf/keras-fractalnet#2]
I have taken a deeper look into your FractalNet implementation, but there is one thing I don't understand. Your [`global_path_arr`](https://github.com/snf/keras-fractalnet/blob/master/src/fractalnet.py#L146) is initialized with a (seeded) numpy random array. This means that during training it will be generated once and then not changed. 

The fact that you seed the `np.random` at the beginning means that even if the random function was called over and over it would always produce the same results.

I tried to implement FractalNet on Cifar from scratch using your implementation as reference and I have run into the same problem, how to generate one path configuration for drop-path per batch.

Could you clarify if this is a bug or if I missed something entirely?

**Comment by snf** [comment]
Hi @mirosval , if you check in [rand_one_in_array](https://github.com/snf/keras-fractalnet/blob/master/src/fractalnet.py#L29), it's using the samplers from Theano and Tensorflow respectively. Those samplers will generate random values for each step, you can "print check" them using the [K.print_tensor](https://keras.io/backend/#print_tensor) identity operator available in Keras.
I authored print_tensor in Keras (using Theano and Tensorflow features) for checking exactly the same as I was not sure it was really sampling new values for each step but it was.
